### PR TITLE
Subnormal value fix in PCG

### PR DESCRIPTION
**Pull request by rfalgout**
This fixes a problem Chris Schroeder was having with subnormal values in PCG.

### CODE REVIEW

**Comment by rfalgout** [comment]
Hi @liruipeng .  This is ready to merge once you approve.  Thanks!

**Comment by liruipeng** [comment]
@rfalgout Approved. Commit 11f56bde634ef2599f8edb194f5b775c30eae20a was also applied into branch `amg-setup`. Thank you!

**Review by liruipeng — APPROVED** [review]

**Pull request merged**
